chevah/github_hooks_server/handler.py

```diff
@@ -17,11 +26,13 @@ class Handler(object):
     RE_TRAC_TICKET_ID = r'\[#(\d+)\] .*'
     RE_REVIEWERS = r'.*reviewers{0,1}:{0,1} @.*'
     RE_NEEDS_REVIEW = r'.*needs{0,1}[\-_]review.*'
+    RE_NEEDS_REVIEW_TWISTED = r'.*(please[ \-]review|review[ \-]please).*'
```

Comment:
hm...can't we write a single regex that will match both ?

Or to make it easier to define, define it as a list of regex

Reply:
I changed it in d00bd67 . Does it look OK?

Reply:
yes. thanks